.github/workflows/release.yml

```diff
@@ -162,12 +188,11 @@ jobs:
     - run: docker load < enarx-${{ matrix.platform.target }}-oci
     # TODO: Attempt to run `enarx info` within the loaded container
 
-  Regenerate-BOM:
-    uses: ./.github/workflows/sbom_manifest_action.yml
+  # TODO: redevelop SBOM generation
```

Comment:
Is there an issue about this? If so, please add a reference. Otherwise, file one and add a reference

Reply:
Referenced in issue #1954